specificatie/kadastraal-onroerende-zaken.yaml

```diff
@@ -197,7 +197,7 @@ components:
           _links:
             $ref: "#/components/schemas/KadastraalOnroerendeZaak_links"
           _embedded:
-            $ref: "#/components/schemas/KadastraalOnroerendeZaak_embedded"
+            $ref: "#/components/schemas/KadastraalOnroerendeZaakEmbedded"
     KadastraalOnroerendeZaak_links:
```

Comment:
KadastraalOnroerendeZaak_links => KadastraalOnroerendeZaakLinks

Reply:
en in regel 68: TypeGerechtigde_enum --> TypeGerechtigdeEnum